src/components/pages/devices/devicesDetail/simpleListDisplay.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class SimpleListDisplay extends Component {
+  render() {
+    const { list } = this.props;
+    if (list) {
+      var itemList = list.map((item, idx) => {
```

Comment:
Use `const` instead of `var`. Basically never use var.

Reply:
ok